SFPortfolioApp/SFPortfolioApp/Modules/CategoryModule/BaseListScene/CategoryTableViewCell/CategoryItemTableViewCell.swift

```diff
@@ -30,6 +31,11 @@ class CategoryItemTableViewCell: UIResizableTableViewCell {
         }
     }
     
+    private func setupAccessibility() {
+        tagsStackView.isAccessibilityElement = true
```

Comment:
Check if the default value of 'isAccessibilityElement' isn't true

Reply:
The default value is _false_ for a standard UIKit controls and is removed here for the stackView.